test/Visualization_Tests/src/Widgets/Table_Widgets_Spec.enso

```diff
@@ -59,6 +60,11 @@ add_specs suite_builder =
 
             labels . should_equal ['Equals', 'Equals (Ignore Case)', 'Between']
 
+        group_builder.specify "works for `geo_distance`" <|
+            choices = [Choice.Option "<Number Value>" "0", Choice.Option "<Expression>" "(expr '[A]')"]
+            expect = [["lat1", Widget.Single_Choice choices Nothing Display.Always]] . to_json
+            Widgets.get_widget_json mock_table .geo_distance ["lat1"] . should_equal expect
```

Comment:
> Some unit tests was written

- this is a unit test
- but it doesn't help with _"falsification"_
- as this code was working all the time

Reply:
> how do you want to test it, IDE guys?

The code modified by you is tested in `app/gui/src/project-view/components/GraphEditor/widgets/WidgetFunction/__tests__/widgetFunctionCallInfo.test.ts` We didn't care about the visualization module because the logic was trivial.

Updating it should be enough for the code. But the _root_ cause of this issue was a bug not in the code, but in our minds, and I cannot think of any testing them except the full product tests.